src/backups2datalad/util.py

```diff
@@ -281,3 +282,19 @@ def yaml_dump(data: Any) -> str:
 
 class UnexpectedChangeError(Exception):
     pass
+
+
+def check_git_annex_version() -> None:
+    # Call this function at the start of subcommand functions rather than in
+    # `main()` so that it doesn't run if a user does `backups2datalad
+    # subcommand --help`
```

Comment:
I bet there should be a way for a common to all subcommands CLI context handling outside of --help, or may be in the top level context handling to do it is there is no --version or --help

But ok for now to get us going